starkinfra/issuingtoken/__issuingtoken.py

```diff
@@ -0,0 +1,146 @@
+from starkinfra.utils import rest
+from starkcore.utils.resource import Resource
+from starkcore.utils.checks import check_datetime, check_date
+
+
+class IssuingToken(Resource):
+    """# IssuingToken object
+    The IssuingToken object displays the information of the tokens created in your Workspace.
+    ## Attributes (return-only):
+    - id [string]: unique id returned when IssuingToken is created. ex: "5656565656565656"
+    - status [string]: current IssuingToken status. ex: "active", "blocked", "canceled", "frozen" or "pending"
+    - card_id [string]: card unique id. ex: "5656565656565656"
+    - wallet_id [string]: wallet id. ex: "google"
+    - wallet_name [string]: wallet name. ex: "GOOGLE"
+    - merchant_id [string]: merchant unique id. ex: "5656565656565656"
+    - external_id [string]: a unique string among all your IssuingToken, used to avoid resource duplication. ex: "DSHRMC00002626944b0e3b539d4d459281bdba90c2588791"
```

Comment:
`a unique string among all your IssuingTokens, used to avoid resource duplication. ex: "DSHRMC00002626944b0e3b539d4d459281bdba90c2588791"`